examples/example_leverage_xarray.py

```diff
@@ -0,0 +1,162 @@
+"""
+Examples of leveraging xarray in combination with trajan
+================================================
```

Comment:
the docs building tend to complain that the `===` are not longer than the title.